src/app/tests/suites/certification/Test_TC_ICDM_3_4.yaml

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Auto-generated scripts for harness use only, please review before automation. The endpoints and cluster names are currently set to default
+
+name: "[TC-ICDM-3.4] ICDCounter persistence with DUT as Server"
+
+PICS:
+    - ICDM.S
+
+config:
+    nodeId: 0x12344321
+    cluster: "Basic Information"
+    endpoint: 0
+
+tests:
+    - label: "Preconditions"
```

Comment:
Which part of this test makes it so it can't be automated?

Reply:
We are currently conducting manual tests for the TE2 event in anticipation of its proximity. Simultaneously, we are in the process of creating new test cases and intend to automate these tests, with the plan to submit a corresponding PR.